### ISSUE TITLE
Main site - Bug - Bots are still active? Weird problem in DB LOG section?

### ISSUE THREAD

**Issue by stgraveyard** [atari-legend/atari-legend#59]
Hi Nico,

I noticed this : 

![image](https://user-images.githubusercontent.com/4339294/101275844-31d55f80-37a9-11eb-81b3-f044b6a65c0c.png)

This is now on the main site at the bottom. 

When I go to legacy to check the cpanel and the dblog section I see this : 

![image](https://user-images.githubusercontent.com/4339294/101275856-4dd90100-37a9-11eb-9c80-68e04d10f387.png)

No new entries in the user table to be found. However, at the old site, I can still click the usernames at the bottom of the screen (maybe we can add this feature also on the main site but only when logged in and when admin? always nice to quickly view a user) : 

![image](https://user-images.githubusercontent.com/4339294/101275885-7c56dc00-37a9-11eb-8c19-3dbc85aaae3b.png)

We clearly see this user was made on 06-12-2020, yet is is not anymore in the log. 

So to sum it up : 
- Do we have a bot yet again
- Why are these entries not anymore in the DBLOG section
- Maybe we can add hyperlinks at the bottom of the page on the main site but only for admins who are loggen in

Cheers,
Maarten

**Comment by nguillaumin** [comment]
- Yes it's quite likely the bots are still here. I wanted to have a look at setting up a captcha
- The changelog I completely overlooked it. I realized 2 weeks ago and I started to work on it but it's not deployed yet. I'll try to get to it today 
- I'll try to add the links as well

**Comment by nguillaumin** [comment]
- I have setup a captcha on the register page so that should prevents bots from registering, hopefully for the last time.
- Changelog is now updated when creating users
- Added little pen icons for admins in the users list in the footer to edit users

**Comment by stgraveyard** [comment]
Simply amazing. Thanks.
